rsts/getting_started.sh.rst

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+:'
+
+.. _gettingstarted:
+
+Getting started
+---------------
+
+.. rubric:: Estimated time to complete: 3 minutes.
+
+
+Prerequisites
+***************
+
+Make sure you have `docker installed <https://docs.docker.com/get-docker/>`__ and `git <https://git-scm.com/>`__ installed, then install flytekit:
+
+Steps
+*****
+
+'
+
+# #. First install the python Flytekit SDK (maybe in a virtual environment) and clone the `flytekit-python-template <https://github.com/flyteorg/flytekit-python-template>`_ repo::
+
+  pip install flytekit
+  git clone git@github.com:flyteorg/flytekit-python-template.git myflyteapp
+  cd myflyteapp
+
+
+' #. The repo comes with a sample workflow, which can be found under ``myapp/workflows/example.py``. The structure below shows the most important files and how a typical flyteapp should be laid out.
+.. note::
+
+    .
+    ├── Dockerfile
+    ├── docker_build_and_tag.sh
+    ├── myapp
+    │   ├── __init__.py
+    │   └── workflows
+    │       ├── __init__.py
+    │       └── example.py
+    └── requirements.txt
+
+You can use pip-compile to build your requirements file. the Dockerfile that comes with this is not GPU ready, but is a simple Dockerfile that should work for most apps.
+'
+The workflow can be run locally simply by running it as a python script - ``note the __main__ at the bottom of the file``::
+
+    python myapp/workflows/example.py
```

Comment:
I got this.
```
python3 myapp/workflows/example.py
2021-06-26 05:25:35,410-flytekit-DEBUG$ Task returns unnamed native tuple <class 'str'>
2021-06-26 05:25:35,411-flytekit-DEBUG$ Task returns unnamed native tuple <class 'str'>
```

May be we can also add expected o/p

Reply:
+1

Reply:
added